compare-evidence-strings/compare.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -eu
+
+echo "Set up environment and parse parameters"
+# To ensure that the sort results are consistent, set the sort order locale explicitly.
+export LC_COLLATE=C
+export LC_ALL=C
+# realpath is required to make the paths work after the working directory change.
+OLD_EVIDENCE_STRINGS=$(realpath "$1")
+NEW_EVIDENCE_STRINGS=$(realpath "$2")
+export OLD_EVIDENCE_STRINGS NEW_EVIDENCE_STRINGS
+mkdir comparison && cd comparison || exit 1
```

Comment:
To avoid having an error when the folder already exists
`mkdir -p comparison && cd comparison || exit 1`

Reply:
This was actually intentional: if the directory exists, we don't want to overwrite its contents, because (1) it may be a directory created by someone else previously and the names just coincide, and (2) this could lead to potentially mixing up results from two comparison runs.

However, I agree that the current setup wasn't ideal. It also didn't allow for running multiple comparisons at the same time. So I've now reimplemented this part properly, with creating a temporary directory and performing all work inside it.

Addressed in: https://github.com/opentargets/ot-release-metrics/pull/12/commits/c51b6fad680b5819b1a537cbc71c9941a8fd99c2